app/src/main/java/com/example/learingrealmandretrofit/api/BaseApi.kt

```diff
@@ -13,4 +15,21 @@ object BaseApi {
         .addConverterFactory(GsonConverterFactory.create())
         .build()
         .create(ApiInterface::class.java)
+
+    fun retrofitHeader(token: String): ApiInterface {
```

Comment:
Remove `val retrofit`, it's no longer used

Reply:
val retrofit used from API fun createUser and fun signIn.  They don't have to be with headers